pom.xml

```diff
@@ -163,6 +190,14 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+
+            <!-- GSON -->
```

Comment:
Remove dependency on GSON (this project already has a dep on a JSON parser)

Reply:
Yeah, removed.